.github/workflows/ci.yaml

```diff
@@ -55,14 +55,14 @@ jobs:
           excludeTitle: 'true' # optional: this excludes the title of a pull request
           checkAllCommitMessages: 'true' # optional: this checks all commits associated with a pull request
           accessToken: ${{ secrets.GITHUB_TOKEN }} # github access token is only required if checkAllCommitMessages is true
-        if: ${{ !github.event.pull_request.merged }} && ${{ !cancelled() }}
+        if: ${{ !cancelled() }}
       - name: Check Verb
         run: .github/scripts/is_verb.sh
         env:
           GITHUB_EVENT_NAME: ${{ github.event_name }}
           BASE_SHA: ${{ github.event.pull_request.base.sha }}
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
-        if: ${{ !github.event.pull_request.merged }} && ${{ !cancelled() }}
+        if: ${{ !cancelled() }}
```

Comment:
### Remove needless pull request checks

The original `github.event.pull_request.title` got the title of the **pull request** - checking that for "Merge pull request" or "Merge branch" did not make sense as not the pull request titles have those messages but the commits.
Now, checking the `event` if it is a merged pull request also doesn't make sense because on the main branch the workflow gets triggered by the `push` event.